aiida/common/links.py

```diff
@@ -76,6 +76,23 @@ class GraphTraversalRules(Enum):
         'call_work_backward': GraphTraversalRule(LinkType.CALL_WORK, 'backward', True, True)
     }
 
+    # For vizualization, everything should be allowed, but the graph should be minimal by default
+    # It's also unknown whether this will be used for ancestors or descendants
```

Comment:
what does this mean?

Reply:
For the first line my guess is that it means "every rule is toggleable and false by default". The second line, I have no idea. But I'm not sure there is a need to justify this with a comment, right? Maybe this could go away. @zooks97 opinions? Also, is there a reason why 'input_calc_backward' has toogable=False or was this a mistake?

Reply:
1. It is an error that `input_calc_backward` has `toggleable=False`. It should instead be true
2. Comment 1 means that for visualization, every traversal rule should be toggleable, but all traversal rules should be `False` by default to give the minimal graph (i.e. only the input node with no links followed).
3. Comment 2 means that there should be no difference in the default traversal rules for visualization between `forward` and `backward` because the same default traversal rules are followed for ancestors and descendants. So, just as in comment 1, all are toggleable but False by default.

Reply:
Ok, I see. If you agree, I think it we could remove the comments since they are mostly stating what can be understood from just looking at the dictionary.

Reply:
Let's replace this with
```
# For visualization allow toggling of all rules, with the default settings yielding a minimal graph
```

Reply:
Changed the name of the `VISUALIZE` rules to `DEFAULT` and wrote requested comment in the graph class when the rules are loaded instead of the links module.